app/config.go

```diff
@@ -144,20 +145,32 @@ func GetProperties(properties []*data.Attribute) (map[string]interface{}, error)
 			if err != nil {
 				return props, err
 			}
-			props[property.Name()] = value
+
+			strVal, ok := value.(string)
+			if ok && strings.HasPrefix(strVal, "SECRET:") {
```

Comment:
@mellistibco : We decided to use "SECRET:" based on industry practice. Though, case sensitivity should not matter, it is consistent with how secrets are serialized in flogo.json.

Since, we support hierarchy, I feel case sensitivity is needed. Though, lets revisit this once we have some feedback from the users.